### PR TITLE
build using newest fccsw

### PR DESCRIPTION
**Pull request by joequant**
Needed to build in gcc10

### CODE REVIEW

File: packages/fccsw/package.py [hunk 1]
```diff
@@ -42,12 +42,12 @@ class Fccsw(CMakePackage):
     depends_on('geant4', when='@0.13:')
     depends_on('geant4@10.6.1', when='@:0.12')
     depends_on('hepmc@:2.99.99')
-    depends_on('heppdt@:2.99.99')
```
Comment (vvolkl) [inline packages/fccsw/package.py:45]: Why this change? I recall getting a build error with the 3.X version, and all the lcg releases use 2.06.01.

**Comment by joequant** [comment]
I'll double check to see if that's still necessary.  One good/bad thing
about the spack release is that since people are putting
new fixes, that something that breaks one week might work the next.

**Comment by vvolkl** [comment]
> I'll double check to see if that's still necessary. One good/bad thing about the spack release is that since people are putting new fixes, that something that breaks one week might work the next.

Thanks! That's true. At one point we might have to pin a version/commit of spack if we want to ensure a reproducible build.

**Comment by joequant** [comment]
Looks like it works now and so the change is unnecessary.
